beacon-chain/sync/pending_blocks_queue.go

```diff
@@ -445,6 +460,16 @@ func (s *Service) addPendingBlockToCache(b interfaces.ReadOnlySignedBeaconBlock)
 	return nil
 }
 
+// This adds blob to slotToPendingBlobs cache.
+func (s *Service) addPendingBlobToCache(b *eth.SignedBlobSidecar) error {
+	blobs := s.pendingBlobsInCache(b.Message.Slot)
+
+	blobs = append(blobs, b)
```

Comment:
Is it possible that we accidentally append a blob that has been already cached, resulting in having the same blob twice?

Reply:
Nice catch! dbdf3851f